views/layouts/main.handlebars

```diff
@@ -10,19 +10,47 @@
 
   </head>
   <body>
-    <nav>
-      <a href='/'>Home</a>
-      {{#if isLoggedIn}}
-        <button class='btn btn-logout'>Log Out</button>
-      {{else}}
-        <a href='/login'>login</a>
-        <a href='/signup'>signup</a>
-      {{/if}}
-    </nav>
-    {{{body}}}
-    {{#if isLoggedIn}}
-      <script src='/js/logout.js'></script>
-    {{/if}}
+
+{{#if isLoggedIn}}
+ <nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <a class="navbar-brand" href="/">Home</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item active">
+          <a class="nav-link" href="/newspot">+ New Spot <span class="sr-only">(current)</span></a>
+        </li>
+        <li class="nav-item active">
+          <a class="nav-link" href="/profile/{{user.id}}">My Spots <span class="sr-only">(current)</span></a>
```

Comment:
You can remove user.id here because /profile will automatically redirect to the user's profile.